src/OrthoTree/OrthoTree.hpp

```diff
@@ -0,0 +1,119 @@
+#include "OrthoTree.h"
+
+namespace ippl {
+
+    template <size_t Dim>
+    OrthoTree<Dim>::OrthoTree(size_t max_depth, size_t max_particles_per_node, const bounds_t& root_bounds)
+        : max_depth_m(max_depth), max_particles_per_node_m(max_particles_per_node),
+        root_bounds_m(root_bounds), morton_helper(max_depth)
+    {
```

Comment:
Why do we need this output to cout?

Do we know how IPPL handles logging?

Reply:
I thought it was cute, will remove it